### PR TITLE
Flash helper with plural name

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Bundle/Resources/config/services/state.xml [hunk 1]
```diff
@@ -13,6 +13,24 @@
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
+        <service id="sylius.resource_factory.expression_language" class="Symfony\Component\ExpressionLanguage\ExpressionLanguage" />
```
Comment (lchrusciel) [inline src/Bundle/Resources/config/services/state.xml]: I fail to see connection of these changes with plural naming

Comment (loic425) [reply src/Bundle/Resources/config/services/state.xml]: Hum indeed, I'll check that.

File: src/Bundle/Resources/config/services/state.xml [hunk 2]
```diff
@@ -13,6 +13,22 @@
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
+        <service id="sylius.resource_factory.argument_parser" class="Sylius\Component\Resource\Factory\ArgumentParser">
+            <argument type="service" id="sylius.resource_factory.expression_language" />
+            <argument type="service" id="request_stack" />
+            <argument type="service" id="security.token_storage" on-invalid="null" />
+        </service>
+        <service id="Sylius\Component\Resource\Factory\ArgumentParserInterface" alias="sylius.resource_factory.argument_parser" />
```
Comment (lchrusciel) [inline src/Bundle/Resources/config/services/state.xml]: Is it needed?
```suggestion
```

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, @loic425!

**Pull request merged**